brian2/synapses/synapses.py

```diff
@@ -1570,3 +1575,17 @@ def _add_synapses_generator(self, j, n, skip_if_invalid=False, namespace=None, l
                                         check_units=False,
                                         run_namespace=namespace)
         codeobj()
+
+    def _check_synapse_parse(self, parsed, namespace):
```

Comment:
I wouldn't worry about this now. It's an internal function (made very clear by starting the name with an underscore), so we can always rename it or change the docstring if the function becomes more general.